docs/tutorials/02-spacy.ipynb

```diff
@@ -73,7 +73,9 @@
    },
    "outputs": [],
    "source": [
-    "%pip install datasets spacy~=3.0 protobuf -qqq"
+    "%pip install datasets spacy~=3.0 protobuf -qqq\n",
+    "%pip install spacy-transformers -f https://download.pytorch.org/whl/torch_stable.html\n",
+    "#install this or %pip install spacy-transformers to avoid problems when loading the model"
```

Comment:
We should explicitly install PyTorch, and then specify `spacy[transformers]` which automatically installs the spacy-transformers package:
```suggestion
    "%pip install torch datasets spacy[transformers]~=3.0 protobuf -qqq",
```